tkltest/util/config_util.py

```diff
@@ -1045,6 +1043,13 @@ def get_modules_properties(tkltest_user_config):
             except subprocess.CalledProcessError as e:
                 tkltest_status('running {} command "{}" failed: {}\n{}'.
                                format(app_build_type, get_modules_command, e, e.stderr), error=True)
+                try:
+                    tkltest_status('command output without --quite:')
```

Comment:
@haim-kermany please change the message to a more coherent one. For example, "Command to obtain modules from user build files failed with the following output:"